lib/puppet_x/voxpupuli/corosync/provider/pcs.rb

```diff
@@ -68,6 +73,20 @@ def self.prefetch(resources)
     end
   end
 
+  def self.run_command_in_cib(cmd, cib = nil, failonfail = true)
+    unless cib.nil?
+      cib_path = File.join(Puppet[:vardir], 'shadow.' + cib)
+      cmd.push('-f', cib_path)
+    end
+    _run_command_in_cib(cmd, cib, failonfail)
+  end
+
+  def self.sync_shadow_cib(cib, failondeletefail = false)
```

Comment:
This creates Pcs::sync_shadow_cib() which is specific for pcs usage. Thus, I would like to move CibHelper::sync_shadow_cib() to Crmsh::sync_shadow_cib() since it is only useful to crmsh now.